src/swapRouter.ts

```diff
@@ -51,8 +55,16 @@ export interface SwapOptions {
    * Optional information for taking a fee on output.
    */
   fee?: FeeOptions
+
+  /**
+   * Optional information for adding liquidity to a position.
+   */
+  addLiquidityOptions?: AddLiquidityOptions
 }
 
+// TODO: must encode self as the recipient to wrap ETH
+const SWAP_ROUTER_ADDRESS = '0x0000000000000000000000000000000000000001'
```

Comment:
ah, this is meant to represent the optimization value indicating `address(this)`? let's use `ADDRESS_THIS` from constants for consistency